config/buses/natss/100-natss.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+data:
+  gnatsd.conf: |
+    # configuration file used to override default NATS server settings
+  stan.conf: |
+    # content of configuration file used to override default NATS Streaming server settings
+kind: ConfigMap
+metadata:
+  labels:
+    app: nats-streaming
+  name: nats-streaming
+  namespace: default
+---  
+apiVersion: v1
+kind: Service
+metadata:
+  name: nats-streaming-headless
```

Comment:
Minor naming comment: I suppose headless is used in case `clusterIP:None`. Please consider removing `headless`.

Reply:
Done